apps/devhub/tests/test_views.py

```diff
@@ -2871,6 +2895,26 @@ def test_unlisted_addon_pass_validation(self, mock_sign_file):
         assert file_.status == amo.STATUS_LITE
         assert mock_sign_file.called
 
+    @mock.patch('devhub.views.sign_file')
+    def test_experiments_are_auto_signed(self, mock_sign_file):
```

Comment:
I might be missing something but I don't see how why this file is accepted, the other test had to give the user permissions.

Frankly this test doesn't make me feel any more confident about the code but I could be missing something that's happening in the base class.